### PR TITLE
Nf gems2 python port

### PR DESCRIPTION
**Pull request by orwonthe**

### CODE REVIEW

**Review by yujanshrestha — APPROVED** [review]

File: python/README.rst [hunk 1]
```diff
@@ -0,0 +1,2 @@
+TODO: proper readme
```
Comment (yujanshrestha) [inline python/README.rst]: We should remove this file for the time being.

**Comment by yujanshrestha** [comment]
Just making sure we did reflect those MATLAB changes that were introduced after we forked off our port correct?

**Comment by orwonthe** [comment]
Changes to matlab code since we forked add some command line options plus an LTA init file. Those can (will) be included when the talairach.lta output is added to atlas registration.

**Comment by yujanshrestha** [comment]
btw @matt-hayden this fork is super old

**Comment by matt-hayden** [comment]
Looks like GitHub helpfully automated bringing it up to the upstream branch

**Pull request merged**